routes/question-mortgage-payments/question-mortgage-payments.controller.js

```diff
@@ -6,7 +6,9 @@ module.exports = (app, route) => {
 
   route.draw(app)
     .get((req, res) => {
-      res.render(name, routeUtils.getViewData(req, {}))
+      res.render(name, routeUtils.getViewData(req, {
+        title: res.__("mortgage_payments.title"),
```

Comment:
small thing, but we're using single quotes elsewhere. Doesn't affect the code, so just leaving as suggestion to change this :)